### DISCUSSION TITLE
Complete scenario for all players in multiplayer

### DISCUSSION THREAD

**Discussion by FlightMS** [OpenRCT2/OpenRCT2#15794] [Feature requests]
In multiplayer mode, only the player which hosts the server will get the achievement when completing the challegene. This should be available for all users on the server.

**Comment by Gymnasiast** [comment]
What are referring to, exactly? OpenRCT2 does not have an achievements system.

**Comment by James103** [comment]
For example, suppose that a multiplayer server hosts a scenario and it finishes with a winning state.
All players on that server would get that scenario marked as complete, instead of just the host.

**Comment by Broxzier** [comment]
OP is probably talking about scenario completion.

**Comment by FlightMS** [comment]
> OP is probably talking about scenario completion.

Yes, that's what I meant here

**Comment by Gymnasiast** [comment]
Changed this to a discussion as I think it’s a matter of preference.
